src/definitions/constants.ts

```diff
@@ -33,6 +35,7 @@ export namespace VSCodeCommands {
   export const ADD_EXTENSIONS = 'quarkusTools.addExtension';
   export const DEBUG_QUARKUS_PROJECT = 'quarkusTools.debugQuarkusProject';
   export const QUARKUS_WELCOME = 'quarkusTools.welcome';
+  export const QUARKUS_OPEN_URL = 'quarkus.open.url';
```

Comment:
Why you need that?

Reply:
Looks like it is some old code from before, I will remove it